native/src/binaryRoutePlanner.cpp

```diff
@@ -15,6 +15,7 @@ static const bool ASSERT_CHECKS = true;
 static const bool PRINT_ROUTING_ALERTS = false;
 static const bool DEBUG_BREAK_EACH_SEGMENT = false;
 static const bool DEBUG_PRECISE_DIST_MEASUREMENT = false;
+static const float MIN_COST_TOLERANCE = 2.0; // required due to imprecise nature of squareRootDist31
```

Comment:
not clear what should be done to avoid it and yet not clear why 2.0 it's not clear. Error 0.02% for squareroot dist.
The fix might be correct but without explanation it might be not enough or incorrect

Reply:
refactored with multiplier (%) and compatibility with 0.1 as the lowest clearance